src/dotnet/APIView/APIViewWeb/Repositories/CosmosAPIRevisionsRepository.cs

```diff
@@ -19,10 +19,12 @@ namespace APIViewWeb
     public class CosmosAPIRevisionsRepository : ICosmosAPIRevisionsRepository
     {
         private readonly Container _apiRevisionContainer;
+        private readonly Container _reviewsContainer;
 
         public CosmosAPIRevisionsRepository(IConfiguration configuration, CosmosClient cosmosClient)
         {
             _apiRevisionContainer = cosmosClient.GetContainer(configuration["CosmosDBName"], "APIRevisions");
+            _reviewsContainer = cosmosClient.GetContainer(configuration["CosmosDBName"], "Reviews");
```

Comment:
There is a cosmosReviewsRepository class: https://github.com/Azure/azure-sdk-tools/blob/main/src/dotnet/APIView/APIViewWeb/Repositories/CosmosReviewRepository.cs 

All the updates should be happening there, so you should move the `UpdateParentReviewLastUpdatedOnAsync` method to that class and call it here, that will also reduce the duplicate implementation of  `public async Task<ReviewListItemModel> GetReviewAsync(string reviewId)
`